fibo/src/views/Ontology.vue

```diff
@@ -1245,6 +1255,20 @@ export default {
           if (body.type !== "details") {
             console.error(`body.type: ${body.type}, expected: details`);
           }
+          // check if resource is deprecated
+          if (body.result.properties.other &&
```

Comment:
it means that you are looking for deprecated in others, @mereolog have to ensure that this field will not be placed in any other group in the configuration and it will always be here

Reply:
Ok, I saw it in 'other' and assumed it's always the case. Should I look for it in every group instead?

Reply:
look [config history](https://github.com/edmcouncil/onto-viewer/commits/develop/onto-viewer-web-app/config/groups_config.xml), `deprecated` should be grouped in ontological characteristic (changes 9 days ago) and now (since the configuration update on the server from yesterday) is grouped like this on all servers. It is necessary to change. earlier I tested with a local instance with old configuration and it worked, now with the newer one I don't get the notification.

Reply:
![image](https://user-images.githubusercontent.com/26776168/156766389-cb506656-7416-40ed-bad6-e39a0b65d573.png)
now with newest config, this look like